docs/README.md

```diff
@@ -0,0 +1,19 @@
+# Documentation writing
+
+## Compile GitHub Pages locally
+### Install required tools and dependencies:
+* [Install ruby 2.x or higher](https://www.ruby-lang.org/en/documentation/installation/)
+* Install bundler:
+    * Execute:`gem install bundler`
+* Install jekyll and its dependencies
+   * Change dir to this directory (`<repository_dir>/docs`) 
+   * Execute `bundle install`
+
+See [here](https://help.github.com/articles/setting-up-your-github-pages-site-locally-with-jekyll) for more information.
+
+### Compile and show documentation
+* Move into this directory (`<repository_dir>/docs`) 
+* Execute `bundle exec jekyll serve`
```

Comment:
Maybe it's my Ruby setup (although I followed the install wizard), but `bundle exec jekyll serve` won't work: "bundler: command not found: jekyll".
Calling `jeykll serve` directly works, though.

Reply:
I tried the manual again. I followed the installation manual and used a fresh docker image ubuntu:18.04. The only challenge is to execute `bundle install`. I had to install the packages `ruby ruby-dev build-essential git libpng-dev zlibc zlib1g zlib1g-dev`.
After executing 'bundle install' in the docs dir everything works fine. Did you use Windows or a Linux-VM?